package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-core",
   "description": "A generic entry point class for FusionJS applications that is used by the FusionJS runtime.",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Technically a breaking change for Flow types, but functionality should be backwards compatible due to the `|| Token` fallback.